README.md

```diff
@@ -16,8 +16,45 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Avoid using `.click()`, `.change()`, et al as event handlers]
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Avoid using `.click()`, `.change()`, et al as event handlers
+### Why?
+Using the .on() method instead can reduce memory usage and works on dynamically added elements.
+
+Events added with .on() can also be namespaced. This make code easier to read and when removing the event you are sure you are not accidentally removing other event handlers.
+
+### How?
+``` javascript
+// Bad
+$('.todo-item').click(function(event) {
+  // ...
+});
+
+// Good
+$('.todo-item').on('click', function() {
+  // ...
+});
+
+// Better
+$('.todo-list').on('click', '.todo-item', function(event) {
```

Comment:
Turn "Use event delegation" into a separate guideline? We can then link to it from here.